internal/provider/schema_gitlab_application_settings.go

```diff
@@ -0,0 +1,3054 @@
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"commit_email_hostname": {
+			Description: "Custom hostname (for private commit emails).",
+			Type:        schema.TypeString,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"container_expiration_policies_enable_historic_entries": {
+			Description: "Enable cleanup policies for all projects.",
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"container_registry_cleanup_tags_service_max_list_size": {
+			Description: "The maximum number of tags that can be deleted in a single execution of cleanup policies.",
+			Type:        schema.TypeInt,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"container_registry_delete_tags_service_timeout": {
+			Description: "The maximum time, in seconds, that the cleanup process can take to delete a batch of tags for cleanup policies.",
+			Type:        schema.TypeInt,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"container_registry_expiration_policies_caching": {
+			Description: "Caching during the execution of cleanup policies.",
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"container_registry_expiration_policies_worker_capacity": {
+			Description: "Number of workers for cleanup policies.",
+			Type:        schema.TypeInt,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"container_registry_token_expire_delay": {
+			Description: "Container Registry token duration in minutes.",
+			Type:        schema.TypeInt,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"package_registry_cleanup_policies_worker_capacity": {
+			Description: "Number of workers assigned to the packages cleanup policies.",
+			Type:        schema.TypeInt,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"deactivate_dormant_users": {
+			Description: "Enable automatic deactivation of dormant users.",
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"default_artifacts_expire_in": {
+			Description: "Set the default expiration time for each job’s artifacts.",
+			Type:        schema.TypeString,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"default_branch_name": {
+			Description: "Instance-level custom initial branch name (introduced in GitLab 13.2).",
+			Type:        schema.TypeString,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"default_branch_protection": {
+			Description: "Determine if developers can push to the default branch. Can take: 0 (not protected, both users with the Developer role or Maintainer role can push new commits and force push), 1 (partially protected, users with the Developer role or Maintainer role can push new commits, but cannot force push) or 2 (fully protected, users with the Developer or Maintainer role cannot push new commits, but users with the Developer or Maintainer role can; no one can force push) as a parameter. Default is 2.",
```

Comment:
@timofurrer  - This one's a bit odd. It feels like we should align to our `valid values are:` syntax here, but the description of the integers is pretty important to the use of the resource here, so I feel like we should leave it the way it is.

WDYT?

Reply:
Yes, let's leave it the way it is.